pages/linux/imgp.md

```diff
@@ -0,0 +1,16 @@
+# imgp
+
+> imgp is a command line image resizer and rotator for JPEG and PNG images. 
+
+- Convert some images and directories:
+
+`imgp -x 1366x1000 ~/downloads ~/specifc.png`
```

Comment:
Could you put `1366x1000` in token syntax (i.e. `{{1366x1000}}`)?

Another thing, could you change the paths to use token syntax as well for each example?
`{{path/to/directory}}` and `{{path/to/file}}`